### PR TITLE
Create SR2024 competition attendance confirmation email

### PR DESCRIPTION
**Pull request by Adimote**
## Summary

Add an email asking teams to give us their attendance numbers and special considerations.

## Checklist

- [x] Has front-matter (`to` and `subject`)
- [x] Compared to a similar email from the previous year
- [x] Ensured there's enough context for a rookie team to understand

### CODE REVIEW

**Comment by andybusse** [comment]
Looks good, as this week is half term for most let's plan to send this either Sunday or Monday

**Comment by PeterJCLaw** [comment]
For reference the previous approximately equivalent email was https://github.com/srobo/team-emails/blob/main/SR2023/2023-03-15-competition-details..md. That contains a lot more details about the event itself, in part because it was closer to the event. I think it's fine that we don't include that same level here.

File: SR2024competition-attendance-confirmation.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+to: Student Robotics 2024
+subject: Extra Details for the SR2024 Competition event
+---
+
+Hi,
+
+As you surely know by now, [the competition](https://studentrobotics.org/events/sr2024/competition/) will be on Saturday 13th April and Sunday 14th April!
+
+We're working hard getting the event ready for you. In order for us to cement our plans, We need to know roughly how large your team is and whether there are any special considerations we need to take for anyone who is coming.
```
Comment (ShadowLord2005) [inline SR2024competition-attendance-confirmation.md:10]: Should the 'W' in 'we need to know roughly' be capitalised?
```suggestion
We're working hard getting the event ready for you. In order for us to cement our plans, we need to know roughly how large your team is and whether there are any special considerations we need to take for anyone who is coming.
```

**Comment by PeterJCLaw** [comment]
I think we should explain a little bit about pits and how they work so that any rookie teams have more context about why we need this information.

File: SR2024competition-attendance-confirmation.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
```
[the remark this answers is not in the document]
Comment (PeterJCLaw) [reply SR2024competition-attendance-confirmation.md]: d5c6f086e9507818de3d45bbb40d2269b56acba1

File: SR2024competition-attendance-confirmation.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+to: Student Robotics 2024
+subject: Extra Details for the SR2024 Competition event
+---
+
+Hi,
+
+As you surely know by now, [the competition](https://studentrobotics.org/events/sr2024/competition/) will be on Saturday 13th April and Sunday 14th April!
```
[the remark this answers is not in the document]
Comment (PeterJCLaw) [reply SR2024competition-attendance-confirmation.md]: ed9c21f47d5094427666fbc372247411c953400d

**Review by RealOrangeOne — APPROVED** [review]

**Pull request merged**